### PR TITLE
Use enforce unique instead of memoize unique

### PR DESCRIPTION
**Pull request by MansurAliKoroglu**
Used `enforce-unique` package for unique usernames in test utils and removed `memoize-unique` function.

https://www.npmjs.com/package/enforce-unique

This change is related with:

Issue: #69 
PR: #116 

## Test Plan

- All tests depends on `createUser` db util should pass.
- DB seeding should pass

## Checklist

- [ ] Tests updated
- [ ] Docs updated

## Screenshots

No need

### CODE REVIEW

File: tests/db-utils.ts [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
+import { UniqueEnforcer } from 'enforce-unique'
 
 export function createUser() {
 	const firstName = faker.person.firstName()
 	const lastName = faker.person.lastName()
 
-	const username = unique({
-		firstName: firstName.toLowerCase(),
-		lastName: lastName.toLowerCase(),
+	const uniqueUsernameEnforcer = new UniqueEnforcer()
```
Comment (kentcdodds) [inline tests/db-utils.ts:9]: If I understand the library properly, we need to create this object outside the createUser function.

**Review by kentcdodds — CHANGES_REQUESTED** [review]
Thanks! One change please.

File: tests/db-utils.ts (hunk 1 above)
Comment (MansurAliKoroglu) [reply tests/db-utils.ts]: Changed

**Review by kentcdodds — APPROVED** [review]
Thanks!

**Pull request merged**